tests/page-objects/default/reports/reports.wdio.page.js

```diff
@@ -333,6 +336,15 @@ const getAutomaticReply = async () => {
   };
 };
 
+const getDetailReportRowContent = async (row) => {
+  const labels =  await detailReportRowContent(row, 'label').map(async label => await label.getText());
+  const values =  await detailReportRowContent(row, 'value').map(async label => await label.getText());
```

Comment:
I expected to see some `await Promise.all()` with those `.map()` but I learned it's not how [wdio's API](https://webdriver.io/docs/api/browser/$$/) works 😅 thanks for teaching me this today